test/python/lvm-test.dockerfile

```diff
@@ -0,0 +1,4 @@
+# rust image with lvm utils installed
```

Comment:
for what purpose is this docker file?

Reply:
As we are using lvm binaries from the code we need those installed in the test setup. This docker file is used to create that image.

Reply:
What I mean is - we don't have docker files anywhere why do we need one here?

Reply:
All the tests are currently using `rust:latest` image for the docker containers. For testing the lvm related changes we need the lvm binaries to be present inside the container, which is not available in `rust:latest` image. So we created another image with this docker file and used it in the tests. 

If we dont want to have the dockerfile, we should choose an image with lvm preinstalled.

Reply:
We don't need the dockerfile, just an image with lvm installed to run the lvm commands from mayastor. Once we push this image to some openebs common repo we can remove the docker file.

Reply:
Tried mounting the `vgs` command related binaries into the docker compose file
```
$ ldd /sbin/vgs
	linux-vdso.so.1 (0x00007ffc561a1000)
	libblkid.so.1 => /lib/x86_64-linux-gnu/libblkid.so.1 (0x00007fc229f7d000)
	libaio.so.1 => /usr/lib/x86_64-linux-gnu/libaio.so.1 (0x00007fc229f78000)
	libselinux.so.1 => /lib/x86_64-linux-gnu/libselinux.so.1 (0x00007fc229f4d000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fc229dfe000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fc229c0c000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fc22a4ef000)
	libpcre2-8.so.0 => /usr/lib/x86_64-linux-gnu/libpcre2-8.so.0 (0x00007fc229b7c000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fc229b74000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fc229b51000)
```
Still lvm commands are not working from the mayastor binary. Should we go with the `lvm-test` image approach as it straightforward? This image won't be built again and again, only needs to be pushed to openebs repo